gossip/service/gossip_service_test.go

```diff
@@ -298,16 +298,19 @@ func TestWithStaticDeliverClientNotLeader(t *testing.T) {
 	}
 	n := 2
 	gossips := startPeers(serviceConfig, n, 0, 1)
+	fmt.Printf("We have %d gossips\n", len(gossips))
```

Comment:
please remove redundant prints in this test.

Reply:
fixed